### PR TITLE
Proposal/pisp tx subscribers

### PR DESCRIPTION
**Pull request by lewisdaly**
- Adds a new `Subscribers` object to the `POST /quotes` body and Transaction object
- Additional data models as required
- Didn't touch any existing proposals, but there may be a need to update them given the age of this proposal

@adrianhopebailie I'm not so experienced with writing docs like these, so any tips or areas for improvement would be greatly appreciated!

### CODE REVIEW

File: proposals/adrianhopebailie-201908.md [hunk 1]
```diff
@@ -562,6 +586,106 @@ In the response the CNP can provide a value date that they commit to, based on
...
+| id                 | 1           | `FspId`            | The id of a participant interested in the transaction                                   |
+| role               | 1           | `SubscriptionRole` | The role of the subscription                                                            |
+
+Where `SubscriptionRole` is a enum of the following values:
+
+| Name               | Description                                | 
+| ------------------ | ------------------------------------------ | 
+| `PAYER`            | The sender of the funds                    | 
+| `PAYEE`            | The recipient of the funds                 | 
+| `INITIATOR`        | The participant initiating the transaction |
```
Comment (elnyry-sam-k) [inline proposals/adrianhopebailie-201908.md]: Hi Lewis, in the API Definition docs, I think especially Patters, Use Cases, the Initiator term is used to identify entities / FSPs that initiate several use cases.. Taking that into consideration, can we use something like "PISP" to be more specific but also avoid ambiguity?

Comment (lewisdaly) [reply proposals/adrianhopebailie-201908.md]: Yeah that's a good point. Thanks @elnyry-sam-k

**Comment by adrianhopebailie** [comment]
This LGTM @lewisdaly I'll merge and then we can get wider comment on the main PR or do you think it will be easier for people to review this here first?

File: proposals/adrianhopebailie-201908.md (hunk 1 above)
Comment (lewisdaly) [reply proposals/adrianhopebailie-201908.md]: Fixed

**Comment by lewisdaly** [comment]
Thanks @adrianhopebailie - lets merge it in so we don't have even more conversations spread out in different places.

**Review by elnyry-sam-k — APPROVED** [review]

**Pull request merged**